From 4e95620cfa24c2b00af6bb5a133897f3ecc42919 Mon Sep 17 00:00:00 2001
From: Imagination Technologies <powervr@imgtec.com>
Date: Tue, 3 Jan 2023 14:17:48 -0800
Subject: [PATCH 090/168] zink: Only expose PIPE_CAP_IMAGE_ATOMIC_FLOAT_ADD if
 we can actually add.

Drivers may expose the ext without the add capability, if they can
load/store/exchange.

Fixes: c32f046ab669 ("zink: export PIPE_CAP_IMAGE_ATOMIC_FLOAT_ADD")
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/20506>
---
 .../drivers/zink/ci/zink-anv-tgl-fails.txt    | 910 ++++++++++++++++++
 1 file changed, 910 insertions(+)

diff --git a/src/gallium/drivers/zink/ci/zink-anv-tgl-fails.txt b/src/gallium/drivers/zink/ci/zink-anv-tgl-fails.txt
index e69de29bb2d..76901167290 100644
--- a/src/gallium/drivers/zink/ci/zink-anv-tgl-fails.txt
+++ b/src/gallium/drivers/zink/ci/zink-anv-tgl-fails.txt
@@ -0,0 +1,910 @@
+dEQP-GLES31.functional.layout_binding.ubo.fragment_binding_array,Fail
+dEQP-GLES31.functional.layout_binding.ubo.fragment_binding_max_array,Fail
+dEQP-GLES31.functional.layout_binding.ubo.fragment_binding_multiple,Fail
+dEQP-GLES31.functional.layout_binding.ubo.vertex_binding_array,Fail
+dEQP-GLES31.functional.layout_binding.ubo.vertex_binding_max_array,Fail
+dEQP-GLES31.functional.layout_binding.ubo.vertex_binding_multiple,Fail
+
+# CTS bug: https://gitlab.khronos.org/Tracker/vk-gl-cts/-/issues/4200
+KHR-GLES31.core.compute_shader.max,Fail
+
+# "Invalid result. Region (0x0). Expected: 0 got 1"
+KHR-GLES31.core.viewport_array.dynamic_viewport_index,Fail
+
+# "Fail, buffer content is not well preserved when age > 0 (Fail)"
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_clear_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_clear_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_render_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_render_even_clear_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_render_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_clear_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_clear_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_clear_even_clear_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_clear_even_render_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_clear_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_clear_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_render_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_render_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.no_resize.odd_render_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_clear_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_clear_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_render_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_render_even_clear_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_render_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_clear_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_clear_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_clear_even_clear_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_clear_even_render_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_clear_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_clear_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_render_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_render_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_after_swap.odd_render_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_clear_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_clear_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_render_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_render_even_clear_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_render_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_clear_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_clear_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_clear_even_clear_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_clear_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_clear_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_clear_even_render_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_clear_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_clear_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_even_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_render_even_clear_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_render_even_clear,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_render_even_none,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_render_even_render_render,Fail
+dEQP-EGL.functional.buffer_age.no_preserve.resize_before_swap.odd_render_render_even_render,Fail
+
+dEQP-EGL.functional.query_context.get_current_context.rgba8888_window,Crash
+
+# https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/20508
+multisample-dEQP-GLES3.functional.multisample.default_framebuffer.depth,Fail
+multisample-dEQP-GLES3.functional.multisample.default_framebuffer.num_samples_line,Fail
+multisample-dEQP-GLES3.functional.multisample.default_framebuffer.num_samples_polygon,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.depth_clear_stencil_write,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.depth_write_depth_clear_stencil_write_stencil_clear,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.depth_write_depth_clear,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.depth_write_stencil_clear,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.depth_write_stencil_write,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.scissor_depth_clear_stencil_write,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.scissor_depth_write_stencil_clear,Fail
+multisample-dEQP-GLES3.functional.occlusion_query.stencil_write,Fail
+multisample-dEQP-GLES31.functional.sample_shading.min_sample_shading.default_framebuffer_color,Fail
+multisample-dEQP-GLES31.functional.sample_shading.min_sample_shading.default_framebuffer_discard,Fail
+multisample-dEQP-GLES31.functional.shaders.multisample_interpolation.sample_qualifier.default_framebuffer,Fail
+multisample-dEQP-GLES31.functional.shaders.sample_variables.sample_id.default_framebuffer,Fail
+multisample-dEQP-GLES31.functional.shaders.sample_variables.sample_mask.discard_half_per_pixel.default_framebuffer,Fail
+multisample-dEQP-GLES31.functional.shaders.sample_variables.sample_mask.discard_half_per_sample.default_framebuffer,Fail
+multisample-dEQP-GLES31.functional.shaders.sample_variables.sample_mask.discard_half_per_two_samples.default_framebuffer,Fail
+
+# Around the time of running these tests there are some warnings from the kernel in dma_resv.c, and at least
+# some failures look like not waiting for rendering to complete.
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1_gles2_gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1_gles2_gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1_gles2_gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1_gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1_gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1_gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles1.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_context.gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1_gles2_gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1_gles2_gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1_gles2_gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1_gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1_gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1_gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles1.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.multi_thread.gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles1.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles1.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles1.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.color_clears.single_context.gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.negative_api.create_pixmap_surface,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles2_gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles2_gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles2_gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_context.gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles2_gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles2_gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles2_gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.multi_thread.gles3.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.single_context.gles2.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.single_context.gles2.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.single_context.gles2.rgba8888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.single_context.gles3.rgb565_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.single_context.gles3.rgb888_pbuffer,Fail
+wayland-dEQP-EGL.functional.render.single_context.gles3.rgba8888_pbuffer,Fail
+
+565-nozs-dEQP-GLES3.functional.color_clear.complex_rgb,Fail
+
+spec@egl_chromium_sync_control@conformance@eglGetSyncValuesCHROMIUM_ust_test,Fail
+spec@egl_chromium_sync_control@conformance@eglGetSyncValuesCHROMIUM_msc_and_sbc_test,Fail
+spec@egl_chromium_sync_control@conformance,Fail
+
+# "GLX_MESA_copy_sub_buffer found in both client and server extension strings, but missing from unified string."
+glx@extension string sanity,Fail
+
+glx@glx_arb_create_context_es2_profile@invalid opengl es version,Fail
+glx@glx_arb_create_context_no_error@no error,Fail
+glx@glx_arb_create_context_robustness@invalid reset notification strategy,Fail
+
+# "Could not create initial indirect-rendering context."
+glx@glx_ext_import_context@free context,Fail
+glx@glx_ext_import_context@get context id,Fail
+
+glx@glx_ext_import_context@get current display,Fail
+glx@glx_ext_import_context@import context- multi process,Fail
+glx@glx_ext_import_context@import context- single process,Fail
+glx@glx_ext_import_context@imported context has same context id,Fail
+glx@glx_ext_import_context@make current- multi process,Fail
+glx@glx_ext_import_context@make current- single process,Fail
+glx@glx_ext_import_context@query context info,Fail
+
+# See also the EGL buffer age failures
+glx@glx-buffer-age,Fail
+glx@glx-buffer-age vblank_mode=0,Fail
+
+glx@glx-make-current,Crash
+glx@glx-multi-window-single-context,Crash
+glx@glx-swap-copy,Fail
+glx@glx-swap-pixmap-bad,Fail
+glx@glx-tfp,Crash
+
+# errors like:
+# X Error of failed request:  BadWindow (invalid Window parameter)
+# ../src/gallium/drivers/zink/zink_resource.c:1215: resource_create: Assertion `res->obj->dt' failed.
+glx@glx-visuals-depth,Crash
+
+# ../src/gallium/drivers/zink/zink_kopper.c:859: zink_kopper_update: Assertion `pres->bind & PIPE_BIND_DISPLAY_TARGET' failed.
+glx@glx-visuals-depth -pixmap,Crash
+
+glx@glx-visuals-stencil -pixmap,Crash
+glx@glx-visuals-stencil,Crash
+
+shaders@glsl-fs-pointcoord,Fail
+
+shaders@point-vertex-id divisor,Fail
+shaders@point-vertex-id gl_instanceid divisor,Fail
+shaders@point-vertex-id gl_instanceid,Fail
+shaders@point-vertex-id gl_vertexid divisor,Fail
+shaders@point-vertex-id gl_vertexid gl_instanceid divisor,Fail
+shaders@point-vertex-id gl_vertexid gl_instanceid,Fail
+shaders@point-vertex-id gl_vertexid,Fail
+
+spec@!opengl 1.0@gl-1.0-edgeflag-quads,Fail
+spec@!opengl 1.0@gl-1.0-edgeflag,Fail
+
+spec@!opengl 1.0@gl-1.0-no-op-paths,Fail
+
+spec@!opengl 1.0@gl-1.0-spot-light,Fail
+
+spec@!opengl 1.0@gl-1.0-swapbuffers-behavior,Fail
+
+spec@!opengl 1.1@depthstencil-default_fb-blit samples=16,Fail
+spec@!opengl 1.1@depthstencil-default_fb-blit samples=2,Fail
+spec@!opengl 1.1@depthstencil-default_fb-blit samples=6,Fail
+spec@!opengl 1.1@depthstencil-default_fb-blit samples=8,Fail
+
+spec@!opengl 1.1@line-aa-width,Fail
+spec@!opengl 1.1@line-flat-clip-color,Fail
+
+spec@!opengl 1.1@polygon-mode-facing,Fail
+spec@!opengl 1.1@polygon-mode-offset,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 0: Expected blue pixel in center,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 1: Expected blue pixel in center,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 2: Expected blue pixel in center,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 2: Expected white pixel on right edge,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 2: Expected white pixel on top edge,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 5: Expected blue pixel in center,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 6: Expected blue pixel in center,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 6: Expected white pixel on right edge,Fail
+spec@!opengl 1.1@polygon-mode-offset@config 6: Expected white pixel on top edge,Fail
+
+# Frontend issue across multiple drivers.
+spec@!opengl 1.0@rasterpos,Fail
+spec@!opengl 1.0@rasterpos@glsl_vs_gs_linked,Fail
+spec@!opengl 1.0@rasterpos@glsl_vs_tes_linked,Fail
+
+spec@!opengl 1.1@linestipple@Line strip,Fail
+spec@!opengl 1.1@linestipple@Line loop,Fail
+spec@!opengl 1.1@linestipple@Factor 2x,Fail
+spec@!opengl 1.1@linestipple@Factor 3x,Fail
+spec@!opengl 1.1@linestipple,Fail
+
+# polygon-mode: glPolygonMode(front=GL_LINE, back=GL_FILL), glCullMode(GL_NONE/GL_FALSE/GL_NO_ERROR) failed
+# At position 0, found prim GL_FILL instead of GL_LINE
+# polygon-mode: glPolygonMode(front=GL_POINT, back=GL_FILL), glCullMode(GL_NONE/GL_FALSE/GL_NO_ERROR) failed
+# At position 1, found prim GL_POINT instead of GL_FILL
+# (and more)
+spec@!opengl 1.1@polygon-mode,Fail
+
+spec@!opengl 1.1@read-front clear-front-first,Fail
+spec@!opengl 1.1@read-front samples=16,Fail
+spec@!opengl 1.1@read-front samples=2,Fail
+spec@!opengl 1.1@read-front samples=4,Fail
+spec@!opengl 1.1@read-front samples=6,Fail
+spec@!opengl 1.1@read-front samples=8,Fail
+spec@!opengl 1.1@read-front,Fail
+
+# depth texturing: zink is returning intensity shadow comparisons and depth samples, regardless of depth mode.
+# Check out the "splat" in zink_compiler.c around !is_new_style_shadow, and the swizzle rewrite for depth
+# textures in zink_create_sampler_view().
+spec@!opengl 2.0@depth-tex-modes-glsl,Fail
+spec@arb_depth_texture@depth-tex-modes,Fail
+spec@arb_fragment_program_shadow@tex-shadow1d,Fail
+spec@arb_fragment_program_shadow@tex-shadow2d,Fail
+spec@arb_fragment_program_shadow@tex-shadow2drect,Fail
+spec@arb_fragment_program_shadow@txp-shadow1d,Fail
+spec@arb_fragment_program_shadow@txp-shadow2d,Fail
+spec@arb_fragment_program_shadow@txp-shadow2drect,Fail
+spec@arb_shader_texture_lod@execution@arb_shader_texture_lod-texgradcube,Fail
+spec@arb_shader_texture_lod@execution@glsl-fs-shadow2dgradarb-01,Fail
+spec@arb_shader_texture_lod@execution@glsl-fs-shadow2dgradarb-03,Fail
+spec@arb_shader_texture_lod@execution@glsl-fs-shadow2dgradarb-07,Fail
+spec@arb_shader_texture_lod@execution@glsl-fs-shadow2dgradarb-08,Fail
+spec@arb_shader_texture_lod@execution@glsl-fs-shadow2dgradarb-cumulative,Fail
+spec@arb_texture_rectangle@glsl-fs-shadow2drect-01,Fail
+spec@arb_texture_rectangle@glsl-fs-shadow2drect-03,Fail
+spec@arb_texture_rectangle@glsl-fs-shadow2drect-07,Fail
+spec@arb_texture_rectangle@glsl-fs-shadow2drect-08,Fail
+spec@arb_texture_rectangle@glsl-fs-shadow2drect,Fail
+spec@arb_texture_rectangle@glsl-fs-shadow2drectproj,Fail
+spec@arb_texture_rg@execution@fs-shadow2d-red-01,Fail
+spec@arb_texture_rg@execution@fs-shadow2d-red-02,Fail
+spec@arb_texture_rg@execution@fs-shadow2d-red-03,Fail
+spec@ext_texture_array@glsl-fs-shadow1darray-01,Fail
+spec@ext_texture_array@glsl-fs-shadow1darray-03,Fail
+spec@ext_texture_array@glsl-fs-shadow1darray-07,Fail
+spec@ext_texture_array@glsl-fs-shadow1darray-08,Fail
+spec@ext_texture_array@glsl-fs-shadow1darray-bias,Fail
+spec@ext_texture_array@glsl-fs-shadow1darray,Fail
+spec@ext_texture_array@glsl-fs-shadow2darray-01,Fail
+spec@ext_texture_array@glsl-fs-shadow2darray-03,Fail
+spec@ext_texture_array@glsl-fs-shadow2darray-07,Fail
+spec@ext_texture_array@glsl-fs-shadow2darray-08,Fail
+spec@ext_texture_array@glsl-fs-shadow2darray,Fail
+spec@ext_texture_swizzle@depth_texture_mode_and_swizzle,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1d-01,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1d-03,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1d-07,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1d-08,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1d-bias,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1d,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1dproj-bias,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow1dproj,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2d-01,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2d-03,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2d-07,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2d-08,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2d-bias,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2d,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2dproj-bias,Fail
+spec@glsl-1.10@execution@samplers@glsl-fs-shadow2dproj,Fail
+
+spec@!opengl 2.0@gl-2.0-edgeflag-immediate,Fail
+spec@!opengl 2.0@gl-2.0-edgeflag,Fail
+
+spec@!opengl 2.0@vs-point_size-zero,Fail
+
+spec@!opengl 2.1@pbo,Fail
+spec@!opengl 2.1@pbo@test_polygon_stip,Fail
+
+spec@!opengl 2.1@polygon-stipple-fs,Fail
+
+spec@!opengl 3.0@clearbuffer-depth-cs-probe,Fail
+
+spec@!opengl 3.2@gl-3.2-adj-prims cull-back pv-first,Fail
+spec@!opengl 3.2@gl-3.2-adj-prims cull-front pv-first,Fail
+spec@!opengl 3.2@gl-3.2-adj-prims line cull-back pv-first,Fail
+spec@!opengl 3.2@gl-3.2-adj-prims line cull-front pv-first,Fail
+spec@!opengl 3.2@gl-3.2-adj-prims pv-first,Fail
+
+spec@!opengl es 2.0@glsl-fs-pointcoord,Fail
+
+spec@arb_arrays_of_arrays@execution@image_store@basic-imagestore-mixed-const-non-const-uniform-index,Fail
+spec@arb_arrays_of_arrays@execution@image_store@basic-imagestore-mixed-const-non-const-uniform-index2,Fail
+spec@arb_arrays_of_arrays@execution@image_store@basic-imagestore-non-const-uniform-index,Fail
+
+spec@arb_framebuffer_object@fbo-blit-scaled-linear,Fail
+
+spec@arb_framebuffer_object@fbo-gl_pointcoord,Fail
+
+spec@arb_gl_spirv@execution@xfb@vs_block_array,Fail
+
+spec@arb_gpu_shader_fp64@execution@arb_gpu_shader_fp64-tf-separate,Fail
+spec@arb_gpu_shader_fp64@execution@glsl-fs-loop-unroll-mul-fp64,Fail
+spec@arb_gpu_shader_fp64@uniform_buffers@fs-ubo-load.indirect.3,Fail
+
+spec@arb_point_sprite@arb_point_sprite-checkerboard,Fail
+spec@arb_point_sprite@arb_point_sprite-mipmap,Fail
+
+spec@arb_program_interface_query@arb_program_interface_query-getprogramresourceindex,Fail
+spec@arb_program_interface_query@arb_program_interface_query-getprogramresourceindex@'vs_input2[1][0]' on GL_PROGRAM_INPUT,Fail
+
+spec@arb_sample_locations@test,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 0- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 1- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 2- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 1- X: 3- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 0- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 1- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 2- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 16- X: 3- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 0- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 1- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 2- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 2- X: 3- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 0- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 1- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 2- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 4- X: 3- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 0- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 1- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 2- Y: 6- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 0- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 0- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 1- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 1- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 2- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 2- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 3- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 3- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 4- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 4- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 5- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 5- Grid: true,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 6- Grid: false,Fail
+spec@arb_sample_locations@test@MSAA: 8- X: 3- Y: 6- Grid: true,Fail
+spec@arb_sample_shading@samplemask 16 all,Fail
+spec@arb_sample_shading@samplemask 16 all@0.062500 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16 all@0.125000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16 all@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16 all@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16 all@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16 all@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16 all@noms partition,Fail
+spec@arb_sample_shading@samplemask 16 all@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16,Fail
+spec@arb_sample_shading@samplemask 16@0.062500 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16@0.125000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 16@noms partition,Fail
+spec@arb_sample_shading@samplemask 16@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2 all,Fail
+spec@arb_sample_shading@samplemask 2 all@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2 all@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2 all@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2 all@noms partition,Fail
+spec@arb_sample_shading@samplemask 2 all@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2,Fail
+spec@arb_sample_shading@samplemask 2@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 2@noms partition,Fail
+spec@arb_sample_shading@samplemask 2@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4 all,Fail
+spec@arb_sample_shading@samplemask 4 all@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4 all@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4 all@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4 all@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4 all@noms partition,Fail
+spec@arb_sample_shading@samplemask 4 all@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4,Fail
+spec@arb_sample_shading@samplemask 4@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 4@noms partition,Fail
+spec@arb_sample_shading@samplemask 4@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6 all,Fail
+spec@arb_sample_shading@samplemask 6 all@0.125000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6 all@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6 all@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6 all@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6 all@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6 all@noms partition,Fail
+spec@arb_sample_shading@samplemask 6 all@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6,Fail
+spec@arb_sample_shading@samplemask 6@0.125000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 6@noms partition,Fail
+spec@arb_sample_shading@samplemask 6@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8 all,Fail
+spec@arb_sample_shading@samplemask 8 all@0.125000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8 all@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8 all@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8 all@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8 all@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8 all@noms partition,Fail
+spec@arb_sample_shading@samplemask 8 all@sample mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8,Fail
+spec@arb_sample_shading@samplemask 8@0.125000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8@0.250000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8@0.500000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8@1.000000 mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8@noms mask_in_one,Fail
+spec@arb_sample_shading@samplemask 8@noms partition,Fail
+spec@arb_sample_shading@samplemask 8@sample mask_in_one,Fail
+
+spec@arb_shader_image_load_store@early-z,Fail
+spec@arb_shader_image_load_store@early-z@occlusion query test/early-z pass,Fail
+spec@arb_shader_image_load_store@host-mem-barrier,Fail
+spec@arb_shader_image_load_store@host-mem-barrier@Transform feedback/WaW/one bit barrier test/16x16,Fail
+spec@arb_shader_image_load_store@host-mem-barrier@Transform feedback/WaW/one bit barrier test/4x4,Fail
+spec@arb_shader_image_load_store@host-mem-barrier@Transform feedback/WaW/one bit barrier test/64x64,Fail
+
+spec@arb_tessellation_shader@execution@variable-indexing@tcs-patch-vec4-swiz-index-wr,Fail
+
+# "../src/gallium/drivers/zink/zink_compiler.c:2071: assign_producer_var_io: Assertion `*reserved < MAX_VARYING' failed."
+spec@arb_tessellation_shader@execution@variable-indexing@tes-both-input-array-float-index-rd,Crash
+spec@arb_tessellation_shader@execution@variable-indexing@tes-both-input-array-vec2-index-rd,Crash
+spec@arb_tessellation_shader@execution@variable-indexing@tes-both-input-array-vec3-index-rd,Crash
+spec@arb_tessellation_shader@execution@variable-indexing@tes-both-input-array-vec4-index-rd,Crash
+
+# "arb_texture_buffer_object-formats: ../src/gallium/drivers/zink/zink_context.c:807: create_bvci: Assertion `bvci.format' failed."
+spec@arb_texture_buffer_object@formats (vs- arb),Crash
+
+spec@arb_texture_buffer_object@formats (fs- arb),Crash
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA16,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA16F_ARB,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA16I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA16UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA32F_ARB,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA32I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA32UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA8,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA8I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_ALPHA8UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA16F_ARB,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA16I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA16UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA32F_ARB,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA32I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA32UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA8I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE_ALPHA8UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE16_ALPHA16,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE16,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE16F_ARB,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE16I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE16UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE32F_ARB,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE32I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE32UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE8_ALPHA8,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE8,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE8I_EXT,Fail
+spec@arb_texture_buffer_object@formats (fs- arb)@GL_LUMINANCE8UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA16,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA16F_ARB,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA16I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA16UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA32F_ARB,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA32I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA32UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA8,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA8I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_ALPHA8UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA16F_ARB,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA16I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA16UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA32F_ARB,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA32I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA32UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA8I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE_ALPHA8UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE16_ALPHA16,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE16,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE16F_ARB,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE16I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE16UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE32F_ARB,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE32I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE32UI_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE8_ALPHA8,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE8,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE8I_EXT,Fail
+spec@arb_texture_buffer_object@formats (vs- arb)@GL_LUMINANCE8UI_EXT,Fail
+
+spec@arb_texture_cube_map@copyteximage cube samples=16,Fail
+spec@arb_texture_cube_map@copyteximage cube samples=2,Fail
+spec@arb_texture_cube_map@copyteximage cube samples=4,Fail
+spec@arb_texture_cube_map@copyteximage cube samples=6,Fail
+spec@arb_texture_cube_map@copyteximage cube samples=8,Fail
+
+spec@egl 1.4@eglterminate then unbind context,Fail
+
+spec@egl_khr_gl_image@egl_khr_gl_renderbuffer_image-clear-shared-image gl_depth_component24,Fail
+
+spec@egl_khr_surfaceless_context@viewport,Fail
+
+spec@egl_mesa_configless_context@basic,Fail
+
+# Fails across most drivers, but it's a silly test.
+spec@ext_framebuffer_blit@fbo-blit-check-limits,Fail
+
+spec@ext_framebuffer_multisample@blit-mismatched-formats,Fail
+spec@ext_framebuffer_multisample@clip-and-scissor-blit 16 msaa,Fail
+spec@ext_framebuffer_multisample@enable-flag,Fail
+spec@ext_framebuffer_multisample@interpolation 16 centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 16 centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 16 non-centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 16 non-centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 2 centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 2 centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 2 non-centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 2 non-centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 4 centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 4 centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 4 non-centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 4 non-centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 6 centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 6 centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 6 non-centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 6 non-centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 8 centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 8 centroid-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 8 non-centroid-deriv-disabled,Fail
+spec@ext_framebuffer_multisample@interpolation 8 non-centroid-disabled,Fail
+
+spec@ext_packed_float@query-rgba-signed-components,Fail
+
+spec@ext_texture_array@copyteximage 1d_array samples=16,Fail
+spec@ext_texture_array@copyteximage 1d_array samples=2,Fail
+spec@ext_texture_array@copyteximage 1d_array samples=4,Fail
+spec@ext_texture_array@copyteximage 1d_array samples=6,Fail
+spec@ext_texture_array@copyteximage 1d_array samples=8,Fail
+spec@ext_texture_array@copyteximage 2d_array samples=16,Fail
+spec@ext_texture_array@copyteximage 2d_array samples=2,Fail
+spec@ext_texture_array@copyteximage 2d_array samples=4,Fail
+spec@ext_texture_array@copyteximage 2d_array samples=6,Fail
+spec@ext_texture_array@copyteximage 2d_array samples=8,Fail
+
+spec@ext_transform_feedback@tessellation triangle_fan flat_first,Fail
+
+spec@glsl-1.50@execution@primitive-id-no-gs-quad-strip,Fail
+spec@glsl-1.50@execution@primitive-id-no-gs-quads,Fail
+
+spec@khr_texture_compression_astc@miptree-gl srgb-fp,Fail
+spec@khr_texture_compression_astc@miptree-gl srgb-fp@sRGB decode full precision,Fail
+spec@khr_texture_compression_astc@miptree-gles srgb-fp,Fail
+spec@khr_texture_compression_astc@miptree-gles srgb-fp@sRGB decode full precision,Fail
+spec@khr_texture_compression_astc@sliced-3d-miptree-gl srgb-fp,Fail
+spec@khr_texture_compression_astc@sliced-3d-miptree-gl srgb-fp@sRGB decode full precision,Fail
+spec@khr_texture_compression_astc@sliced-3d-miptree-gles srgb-fp,Fail
+spec@khr_texture_compression_astc@sliced-3d-miptree-gles srgb-fp@sRGB decode full precision,Fail
+
+# https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/20506
+spec@nv_shader_atomic_int64@execution@shared-atomicadd-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicand-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicand-uint,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomiccompswap-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicexchange-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicmax-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicmax-uint,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicmin-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicor-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicor-uint,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicxor-int,Fail
+spec@nv_shader_atomic_int64@execution@shared-atomicxor-uint,Fail
-- 
2.17.1

